spec/factories/tasks.rb

```diff
@@ -5,5 +5,11 @@
     from = Date.parse("2019/08/01")
     to   = Date.parse("2019/12/31")
     deadline { Random.rand(from..to) }
+
+    trait :done do
+      status { 2 }
```

Comment:
enumの値は固定値だと伝わらないのでSymbolを使って指定しましょう！
```suggestion
      status { :done }
```